_signal_processing/krakenSDR_signal_processor.py

```diff
@@ -239,6 +241,46 @@ def resetPeakHold(self):
         if self.spectrum_fig_type == "Single":
             self.peak_hold_spectrum = np.ones(self.spectrum_window_size) * -200
 
+    def calculate_squelch(self, sampling_freq, N):
+        def is_enabled_auto_squelch(v):
+            return v == "Auto" or (v == "Default" and self.vfo_default_squelch_mode == "Auto")
+
+        def find_nearest(array, value):
+            array = np.asarray(array)
+            idx = (np.abs(array - value)).argmin()
+            return idx, array[idx]
+
+        if self.spectrum_fig_type == "Single":
+            spectrum_index = 1
+        else:
+            spectrum_index = 2
+        sensor1_spec = self.spectrum[spectrum_index, :]
```

Comment:
@redradist why is it called `sensor1`?

Reply:
To indicate that it is spectrum of RF channel 1 (not 0)

Reply:
I think the term `sensor` is never used in the code and mentioning a number is unnecessary. Instead, in my opinion, it would be nice to name it according to the purpose.

Reply:
Removed in last commit